src/provider/cloud-element-templates/util/propertyUtil.js

```diff
@@ -188,6 +191,7 @@ export function setPropertyValue(bpmnFactory, commandStack, element, property, v
 
   // property
   if (type === 'property') {
```

Comment:
Potential improvement: Get rid of magic value.

Reply:
done in [13c19df](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/763/commits/13c19df796f42a585f7204fea25678b75ae70f9c)